README.md

```diff
@@ -88,12 +96,13 @@ Target //beacon-chain:beacon-chain up-to-date:
 
 In the example above, the beacon chain binary has been created in `bazel-bin/beacon-chain/linux_amd64_stripped/beacon-chain`.
 
-## Running an Ethereum 2.0 Beacon Node
+## Running a beacon node
```

Comment:
+1 on the comment above, create a section called `Connecting to the Public Testnet` with brief mentions about how these are instructions to run a beacon+validator connected to our public eth2 testnet